Готовая работа под Абсолют (для секции about)

YevhenBez opened the pull request:
Готовая работа под Абсолют

YevhenBez committed:
Готовая работа под Абсолют

Готовая работа под Абсолют

RomarioNik commented on src/sass/components/_section-about.scss:
```diff
@@ -0,0 +1,367 @@
+}
+
+.about__text {
+    font-family: Montserrat;
```
font-family: Montserrat;
этот шрифт идет по умолчанию, указывать не обязательно

RomarioNik commented on src/sass/components/_section-about.scss:
```diff
@@ -0,0 +1,367 @@
+.about {
+    padding-top: 61px;
+    padding-bottom: 64px;
```
padding-bottom: 64px;
нижний отступ на секции не нужен, он есть в соседней секции

RomarioNik commented on src/partials/section-about.html:
```diff
@@ -1,5 +1,112 @@
+        >
+        <img
+          src="./images/cow.jpg" 
+          alt="корова" 
```
alt="корова"
Язык сайта английский, alt лучше писать на английском

RomarioNik commented on src/sass/components/_section-about.scss:
```diff
@@ -0,0 +1,367 @@
+}
+
+.about__mood-text {
+    font-family: Montserrat;
```
по умолчанию идет

RomarioNik commented on src/sass/components/_section-about.scss:
```diff
@@ -0,0 +1,367 @@
+
+    background-color: var(--accent-text-color);
+
+    font-family: Montserrat;
```
по умолчанию идет

YevhenBez committed:
Версия 26.12.2022г Секция about

В этой версии нет: 1) Необходимо накачать ещё фоток для версии befor   и   after; 2) Не учтены пожелания Романа, а именно: "......
По поводу текста. У вас три блока текста в одной обертке. И у всех трех блоков есть одинаковые стили. Одинаковые стили лучше вынести в общий класс, а конкретному блоку в отдельный. Например: width: 220px;. Чтобы его изменить нужно искать по всем блокам.

Roman, [26.12.2022 10:26]
И текстовому блоку лучше на 320рх задать боковые паддинги по 30рх и сделать width: 100%, и пусть он тянется до 480рх. А на 480рх зафиксировать его на ширине 440рх.

Zerossik committed:
beta

Zerossik committed:
bg add

Zerossik committed:
img add

Zerossik committed:
cow add

Zerossik committed:
mobile done

Zerossik committed:
tablet done

Zerossik committed:
retina done

Zerossik committed:
html fix

RomarioNik committed:
Merge branch 'main' into about_ice-cream
